detector/aws_instance_previous_type.go

```diff
@@ -6,7 +6,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstancePreviousType(issues *[]*issue.Issue) {
+type AwsInstancePreviousTypeDetector struct {
```

Comment:
__[gometalinter]__
* [golint] exported type AwsInstancePreviousTypeDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->